test/support/webhook_plug.ex

```diff
@@ -0,0 +1,28 @@
+defmodule WebHookPlug do
+  @moduledoc false
+  import Plug.Conn
+  alias Jellyfish.ServerMessage
+  alias Phoenix.PubSub
+
+  @pubsub Jellyfish.PubSub
+
+  def init(opts) do
+    # initialize options
+
+    opts
+  end
+
+  def call(conn, _opts) do
+    {:ok, body, conn} = Plug.Conn.read_body(conn, [])
+    notification = Jason.decode!(body)
+
+    notification =
+      notification |> Map.get("notification") |> ServerMessage.decode() |> Map.get(:content)
+
+    :ok = PubSub.broadcast(@pubsub, "webhook", notification)
```

Comment:
Instead of broadcasting `{:peer_crashed, %PeerCrashed{}}`, let's broadcast `{:webhook_notification, %PeerCrashed{}}`.